pkg/client/deploymentlogs.go

```diff
@@ -34,15 +35,27 @@ func newDeploymentLogs(c *Client, namespace string) *deploymentLogs {
 
 // Get gets the deploymentlogs and return a deploymentLog request
 func (c *deploymentLogs) Get(name string, opts api.DeploymentLogOptions) *kclient.Request {
-	req := c.r.Get().Namespace(c.ns).Resource("deploymentConfigs").Name(name).SubResource("log")
+	req := c.r.Get().Namespace(c.ns).Resource("deploymentConfigs").Name(name).SubResource("log").
+		Param("follow", strconv.FormatBool(opts.Follow)).
+		Param("timestamps", strconv.FormatBool(opts.Timestamps))
```

Comment:
> Open an issue in kube and in origin to fix the serialization of objects to query parameters

But is there any issue really? https://github.com/kubernetes/kubernetes/blob/master/pkg/conversion/queryparams/doc.go#L17-L18

How about converting the object here to a versioned and then converting to params?